### ISSUE TITLE
[US - DEV] Link Users to Comments with Database Relation

### ISSUE THREAD

**Issue by dbradham** [dev-launchers/dev-launchers-platform#1684]
Currently, the only reference of an author on a comment object is their name as plain text. This is a problem for a number of reasons.

1. If two users have the same name we cannot differentiate who submitted the comment.
2. If we want to send a notification when someone replies to a given user's comment, we need to query existing users based on their name
3. If a user ever changes their name, the relationship "breaks"

## Acceptance Criteria
Establish a database relationship between Comments and Users similar to Ideas and Users

**Comment by JulieMass** [comment]
Hey @RETprojects, if you're on this task right now, can you please update its status and move it to the "work in progress" column? Thanks!

**Comment by JulieMass** [comment]
For history documentation: As discussed today with @RETprojects  and @dbradham in Discord, we are moving this ticket to the icebox so Remi can focus on his current bug #953. We’ll come back to this one later. Thank you Remi!

**Comment by RETprojects** [comment]
Everything involving the display names seems to be working properly in staging, except when I go to a workshopping page with comments that were posted before we reworked the storage of users' names on comments, I get a 404 Page Not Found error. I encountered these comments in localhost, and they don't have the proper information stored in them. I believe that if these earlier comments were deleted or possibly edited to reflect the current comment structure, then this 404 error would be resolved and new comments could be posted on these pages.
